utils/update-consul-conf

```diff
@@ -376,6 +377,7 @@ append_confd_svc() {
                         \"--host\", \"$host\",
                         \"--conf-dir\", \"$conf_dir\" ],
             \"interval\": \"1s\",
+            \"timeout\": \"30s\",
```

Comment:
Why we are introducing this timeout?

Reply:
So the probe should wait for this much time before it declares the process as failed. Earlier it was set to default now we are setting it explicitly for better understanding.